python/plugins/processing/gui/AlgorithmDialog.py

```diff
@@ -171,6 +171,29 @@ def runAlgorithm(self):
             self.feedback.pushInfo('')
             start_time = time.time()
 
+            # Formating elapsed time
+            def elapsed_time(start_time, result):
+                delta_t = time.time() - start_time
+                hours = int(delta_t / 3600)
+                minutes = int((delta_t % 3600) / 60)
+                seconds = delta_t - hours*3600 - minutes*60
+
+                str_hours = ["hour", "hours"][hours > 1]
+                str_minutes = ["minute", "minutes"][minutes > 1]
+                str_seconds = ["second", "seconds"][seconds != 1]
```

Comment:
These strings need to be translated:
```suggestion
                str_hours = [self.tr("hour"), self.tr("hours")][hours > 1]
                str_minutes = [self.tr("minute"), self.tr("minutes")][minutes > 1]
                str_seconds = [self.tr("second"), self.tr("seconds"])[seconds != 1]
```